apps/admin-dashboard/app/routes/_dashboard.onboarding-sessions.tsx

```diff
@@ -83,10 +87,12 @@ async function listOnboardingSessions({
       'onboardingSessions.id'
     )
     .leftJoin('students', 'students.id', 'onboardingSessionAttendees.studentId')
+    .leftJoin('admins', 'admins.id', 'onboardingSessions.uploadedById')
     .select([
       'onboardingSessions.date',
       'onboardingSessions.group',
       'onboardingSessions.id',
+      ambassadorNameAggregation,
```

Comment:
The `sql` utility is mainly an escape hatch for things that aren't built into Kysely, so if we can avoid using it I think that's best. The reason this is a little different than the attendees aggregation is because we needed to fetch multiple attendee records and group them into one column, as opposed to the uploaded by field which is not multiple records.

We can simply do the following, which will be type-safe:

```ts
'admins.firstName',
'admins.lastName',
```

Or if you'd like to rename the columns, you can do this which will still be type-safe:

```ts
'admins.firstName as ambassadorFirstName',
'admins.lastName as ambassadorLastName',
```